pkg/pool-stable-phantom/test/StablePhantomPool.test.ts

```diff
@@ -1388,7 +1422,7 @@ describe('StablePhantomPool', () => {
               await pool.swapGivenIn({ in: pool.bpt, out: tokens.first, amount, from: lp, recipient });
 
               const currentBalance = await pool.balanceOf(protocolFeesCollector.address);
-              const expectedFee = getExpectedProtocolFee(amount, AmountKind.WITH_FEE, inRecoveryMode);
+              const expectedFee = getExpectedProtocolFee(amount, AmountKind.WITHOUT_FEE, inRecoveryMode);
```

Comment:
I'm surprised that we had to change this, given the test bellow is quite lax when it comes to how much error it tolerates